### ISSUE TITLE
[BUG] GUI not working properly in LMMS

### ISSUE THREAD

**Issue by tunalad** [yokemura/Magical8bitPlug2#20]
**Describe the bug**
Drop-down instantly closes itself when clicked on, flickers if the mouse is held down.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open with VeSTige
2. Try to open OSC Type/Color/Duty menu
4. Menu flickers

**Expected behaviour**
Being able to change the options that drop-down menus provide.

**Your environment (please complete the following information):**
 - OS: Windows 10 Pro 2004
 - Host DAW and version: LMMS 1.2.2
 - Version of Magical 8bit Plug 2: 1.0.1
 - Plugin format of Magical 8bit Plug you used: VST

**Additional context**
Functions work fine and all while controlling VST from LMMS host (wrench icon, mainly used to automate stuff).
VTS3 is not (yet) supported by LMMS, so can't test that.

**Comment by yokemura** [comment]
Thanks for your report.

I've googled with the keyword `LMMS plugin window pulldown closes immediately` and found this post.
https://lmms.io/forum/viewtopic.php?t=30574#p63238

Is the behavior the same with yours?
If that's the case it is possible that it's the issue of LMMS.

**Comment by tunalad** [comment]
Yup, pretty much the same. I've never experienced that bug in like 3 years of using LMMS so I assumed it was the VST.

**Comment by GrantWilliam5** [comment]
I had this same issue and I found a workaround. If you right click the dropdown menu you can use the arrow keys to select the different options.

hope this helped.

**Comment by Corberius** [comment]
Another workaround is to disable the plugin embedding in the settings of LMMS:

`Edit > Settings > General settings > PLUGIN EMBEDDING > No embedding`

This also fixes the Custom Envelopes not showing when the Advanced Options are enabled in the plugin.

**Comment by yokemura** [comment]
Let me close this issue as it looks like it's the issue of LMMS.
